wysadzulice/assets/js/new_campaign.js

```diff
@@ -0,0 +1,96 @@
+import jquery from 'jquery';
+import Planting from 'plantingjs';
+import { Model, View } from 'backbone';
+
+
+const NewCampaignMapsView = View.extend({
+
+  initialize: function(options) {
+    this.parent = options.parent;
+    this.engine = new Planting({
+      container: this.$el,
+      manifestoUrl: options.manifestoUrl,
+      googleApiKey: options.googleApiKey,
+      saveUrl: this.parent.saveUrl,
+      selectPanoMode: true,
+      onSelectPano: this.parent.on_select_pano.bind(this.parent),
+    });
+  },
+
+});
+
+
+const ItemView = View.extend({
+
+  initialize: function(options) {
+    this.$el.html('<img src="' + options.image + '"></img>');
```

Comment:
Do we need separate view for rendering img tag? Also with ES6 templates you don't need to do string concatenation manually.

```javascript
this.$el.html(`<img src="${options.image}"></img>`);
```